torch/onnx/symbolic_opset9.py

```diff
@@ -1903,21 +1903,28 @@ def new_full(g, self, size, fill_value, dtype, layout, device, pin_memory=False)
 
 
 def eye(g, *args):
-    if len(args) == 5:
-        # aten::eye(n, dtype, layout, device, pin_memory)
-        n, dtype, layout, device, pin_memory = args
+    n, m, k, dtype, layout, device, pin_memory = args
+    is_m_none = m.node().kind() == "prim::Constant" and m.type().kind() == "NoneType"
```

Comment:
Is the check for `node().kind()` necessary if `type().kind() == "NoneType"`?
If it is not, then can we remove the check for `node().kind()`?

Same for the heck below on `is_k_none`

Reply:
It does unfortunately seem to be required.